examples/vulkan/Vulkan.cpp

```diff
@@ -1801,14 +1801,16 @@ class VulkanExample
     void setupTextureImage()
     {
         // Load the image data
-        sf::Image imageData;
+        const auto maybeImageData = sf::Image::loadFromFile("resources/logo.png");
 
-        if (!imageData.loadFromFile("resources/logo.png"))
+        if (!maybeImageData)
```

Comment:
```suggestion
        if (!maybeImageData.has_value())
```

Personal preference, but it provides local context about `maybeImageData` being a `std::optional`.